src/components/internal/OverlayTrigger.tsx

```diff
@@ -76,6 +78,7 @@ export function OverlayTrigger(props: OverlayTriggerProps) {
           disabled={disabled}
           tooltip={tooltip}
           onClick={noop}
+          xss={Css.if(showActiveBorder && state.isOpen).bLightBlue600.$}
```

Comment:
Can we avoid `xss` here and just always supply the `bLightBlue600` if `showActiveBorder` is true? I'm also curious if this border color is the same as the focus ring we'd typically do on Buttons? If it is, maybe we can just use the same focus ring that we already have and force the focus ring to show. Might be worth chatting with design so we don't have multiple ways of displaying a focus sort of style.

Reply:
I'm thinking we'd do something more like:
```
<Button showActiveBorder={state.isOpen} />
or
<Button forceFocusStyles={state.isOpen} />
```

Reply:
I see you updated to just use the focus styles. Was that something approved by Design?

Reply:
Yes, it was approved by Lauren